DeepSpeech.py

```diff
@@ -412,10 +412,21 @@ def try_loading(session, saver, checkpoint_filename, caption):
 
 
 def train():
+    do_cache_dataset = True
+
+    # pylint: disable=too-many-boolean-expressions
+    if    FLAGS.data_aug_features_multiplicative > 0   \
```

Comment:
nit: no spaces between`if` and `FLAGS`, and move `or`s to the end of the lines. And use parenthesis instead of escaping newlines.